zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/archiver/Archiver.java

```diff
@@ -25,6 +32,7 @@
 import org.slf4j.Logger;
 
 public final class Archiver implements CloseableSilently {
+  private static final int BATCH_OPERATION_PARTITION_ID = 1;
```

Comment:
:x: We should use `Protocol.START_PARTITION_ID` to be safe. I don't think we will change in the future, but better safe than sorry, and it's a very small change.